### PR TITLE
Push to Master: Parameter Sweep Scripts

### PR DESCRIPTION
**Pull request by gwaygenomics**
Here are python scripts coding the logic that enables a parameter sweep. This includes:

1. `scripts/vae_paramsweep.py` - submits the 320 jobs to PMACS
2. `scripts/bsub_helper.py` - adds custom configuration to the PMACS job submission
3. `config/*` - assigns the specific parameter sweep and PMACS configuration
4. `parameter_sweep.md` - describes the parameter sweep results
5. `figures/param_sweep/*` - figures summarizing the results of the parameter sweep

In the next PR, I will add the actual parameter sweep text files in the `param_sweep` folder. 
In the PR after that I will add scripts that summarize and visualize the results (although the figures are added here to enable the markdown rendering)

### CODE REVIEW

File: scripts/bsub_helper.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        if self.queue == 'gpu':
+            command_string = (
+                '{} -R "select[ngpus>{}] rusage [ngpus_shared={}]"'
+                .format(command_string, self.num_gpus, self.num_gpus_shared)
+                )
+        command_string = '{} {}'.format(command_string, ' '.join(self.command))
+        return command_string
+
+    def submit_command(self):
+        import subprocess
```
Comment (danich1) [inline scripts/bsub_helper.py]: Why subprocess and not just call os.system?

Comment (gwaygenomics) [inline scripts/bsub_helper.py]: yeah, good question. I'm using them interchangeably. From what I can gather, `subprocess.call()` is just newer and therefore generally preferred to `os.system()`. There are several explanations [out there](https://stackoverflow.com/questions/4813238/difference-between-subprocess-popen-and-os-system), but none give a very satisfying explanation.

Comment (danich1) [inline scripts/bsub_helper.py]: Fair enough. I was just curious as I'm use to using `os.system()` command. Nonetheless, I guess there isn't an issue so either way works.

**Comment by gwaygenomics** [comment]
> I have some questions about the plots, but I'm going to hold those for the next PR

yeah, in retrospect, the figures may have belonged better in the PR with the scripts that generated them, rather than in a PR that made the markdown document more clear

**Pull request merged**